src/storage/config_manager.rs

```diff
@@ -19,20 +19,26 @@ use crate::{
     storage::{lock_manager::LockManager, txn::flow_controller::FlowController, TxnScheduler},
 };
 
-pub struct StorageConfigManger<E: Engine, L: LockManager> {
-    tablet_factory: Arc<dyn TabletFactory<E::Local> + Send + Sync>,
+pub struct StorageConfigManger<E: Engine, EL: engine_traits::KvEngine, L: LockManager> {
```

Comment:
Why?

Reply:
There are two arguments in this structure:

1. `tablet_factory` accepts an instance of `KvEngineFactory` which impls `TabletFactory<engine_rocks::Rocks>,` so `E::Local` should be `engine_rocks::RocksEngine`.
1. `scheduler` is in `Storage` which has type parameter `EK: KvEngine` which is `engine_tiflash::RocksEngine` in our case. So `E::Local` should be `engine_tiflash::RocksEngine`.

So there is a conflict here, so I want to decouple these constraints.